packages/typespec-python/test/azure/generated/azure-mgmt-spheredpg/azure/mgmt/spheredpg/models/_enums.py

```diff
@@ -171,5 +171,5 @@ class UpdatePolicy(str, Enum, metaclass=CaseInsensitiveEnumMeta):
 
     UPDATE_ALL = "UpdateAll"
     """Update all policy."""
-    NO3RD_PARTY_APP_UPDATES = "No3rdPartyAppUpdates"
+    NO3_RD_PARTY_APP_UPDATES = "No3rdPartyAppUpdates"
```

Comment:
@msyyc overall looks good! Is there anyway you can do a quick run through and see if it's breaking any of the existing generations that we don't want to break?

Reply:
Currently we have no similar pipeline do regenerate existing packages with latest main branch. I make an issue to track it:   https://github.com/Azure/autorest.python/issues/2659